recipes/thrust/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package ${CONAN_LIBS})
+set_target_properties(test_package PROPERTIES CXX_STANDARD 11)
+target_compile_definitions(test_package PRIVATE THRUST_DEVICE_SYSTEM=THRUST_DEVICE_SYSTEM_TBB)
```

Comment:
THRUST_DEVICE_SYSTEM should be an option, it will not affect id, because it will be a definition in cpp_info.
```suggestion
```